kuka_mobile_robot_support/urdf/DifferentialMR.urdf

```diff
@@ -0,0 +1,280 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<robot name="genericMR_37146123288">
+<joint name="generic_mr_body_dummy" type="fixed">
+      <parent link="generic_mr_body_dummy" />
+      <child link="generic_mr_body" />
+      <origin xyz="0 0 0" rpy="0 0 0" />
+    </joint>
+  <joint name="caster_wheel_br_joint" type="continuous">
+    <axis xyz="0.0 1.0 0.0"/>
+    <child link="caster_wheel_br_link"/>
+    <origin rpy="-0.0 0.0 -0.0" xyz="-0.015 0.0 -0.026"/>
+    <parent link="caster_base_br_link"/>
+  </joint>
+  <joint name="left_wheel_motor" type="continuous">
+    <axis xyz="0.0 1.0 0.0"/>
+    <child link="left_wheel"/>
+    <limit effort="5.0E2" velocity="1.0E2"/>
+    <origin rpy="-0.0 0.0 -0.0" xyz="0.0 0.325 0.06"/>
+    <parent link="generic_mr_body"/>
+  </joint>
+  <joint name="right_wheel_motor" type="continuous">
+    <axis xyz="0.0 1.0 0.0"/>
+    <child link="right_wheel"/>
+    <limit effort="5.0E2" velocity="1.0E1"/>
+    <origin rpy="-0.0 0.0 -0.0" xyz="0.0 -0.325 0.0"/>
+    <parent link="generic_mr_body"/>
+  </joint>
+  <joint name="caster_base_fl_joint" type="continuous">
+    <axis xyz="0.0 0.0 1.0"/>
+    <child link="caster_base_fl_link"/>
+    <origin rpy="-0.0 0.0 -0.0" xyz="0.48 0.27 0.055"/>
+    <parent link="generic_mr_body"/>
+  </joint>
+  <joint name="caster_base_rl_joint" type="continuous">
+    <axis xyz="0.0 0.0 1.0"/>
+    <child link="caster_base_rl_link"/>
+    <origin rpy="-0.0 0.0 -0.0" xyz="0.48 -0.27 0.055"/>
+    <parent link="generic_mr_body"/>
+  </joint>
+  <joint name="caster_base_bl_joint" type="continuous">
+    <axis xyz="0.0 0.0 1.0"/>
+    <child link="caster_base_bl_link"/>
+    <origin rpy="-0.0 0.0 -0.0" xyz="-0.48 0.27 0.055"/>
+    <parent link="generic_mr_body"/>
+  </joint>
+  <joint name="caster_base_br_joint" type="continuous">
+    <axis xyz="0.0 0.0 1.0"/>
+    <child link="caster_base_br_link"/>
+    <origin rpy="-0.0 0.0 -0.0" xyz="-0.48 -0.27 0.055"/>
+    <parent link="generic_mr_body"/>
+  </joint>
+  <joint name="caster_wheel_fl_joint" type="continuous">
+    <axis xyz="0.0 1.0 0.0"/>
+    <child link="caster_wheel_fl_link"/>
+    <origin rpy="-0.0 0.0 -0.0" xyz="-0.015 0.0 -0.026"/>
+    <parent link="caster_base_fl_link"/>
+  </joint>
+  <joint name="caster_wheel_rl_joint" type="continuous">
+    <axis xyz="0.0 1.0 0.0"/>
+    <child link="caster_wheel_rl_link"/>
+    <origin rpy="-0.0 0.0 -0.0" xyz="-0.015 0.0 -0.026"/>
+    <parent link="caster_base_rl_link"/>
+  </joint>
+  <joint name="caster_wheel_bl_joint" type="continuous">
+    <axis xyz="0.0 1.0 0.0"/>
+    <child link="caster_wheel_bl_link"/>
+    <origin rpy="-0.0 0.0 -0.0" xyz="-0.015 0.0 -0.026"/>
+    <parent link="caster_base_bl_link"/>
+  </joint>
```

Comment:
we should consider combining all of the links and removing these joints, as the robot can only move using the world-base transform

Reply:
ok as it is